docs/Starting Documentation.md

```diff
@@ -0,0 +1,19 @@
+Cairo Vm
+===
+
+# Must Read
+
+* From Cairo Documentation: [How Cairo Works](https://www.cairo-lang.org/docs/how_cairo_works/index.html#how-cairo-works)
+
+* [CairoVm code](https://github.com/starkware-libs/cairo-lang/tree/b614d1867c64f3fb2cf4a4879348cfcf87c3a5a7/src/starkware/cairo/lang/vm)
```

Comment:
Please, change the link to use the master branch:
https://github.com/starkware-libs/cairo-lang/tree/master/src/starkware/cairo/lang/vm